blog/_posts/2016-10-12-Hackbright.md

```diff
@@ -0,0 +1,123 @@
+---
+ title:
+ layout: post
+ created_at: Fri Oct 12 2016
+ permalink: blog/2016-09-30-Hackbrighter-End-of-Summer
+ current: blog
+ author: Melissa Fabros and Patricia Arbona
+ twitter: HeartRGSoC_sf
+ categories:
+ - student-posts
+ ---
+ # Team Hackbrighter's Amazing Summer of Code Experience
+
+ ##### *What accomplishments are you proud of?*
+ ![Patricia Setting Up her Development Environment at Github](/img/blog/2016/hackbrighters_patricia_setup.jpg)<i>Patricia hacking away on a kata</i><br><font color="grey"><small><i>(Image Credit: <a href="https://www.linkedin.com/in/melissa-fabros-5a1b35b">Melissa Fabros</a>)</i></small></font>
+
+ Overall, we're very proud of the fact that our team was able to set up and configure integration testing. According to Martin Fowler's [pyramid of testing](http://martinfowler.com/bliki/TestPyramid.html), integration testing is the hardest and most expensive tests to write. We tried tackling this because we wanted the challenge of learning the codebase at all levels of the stack. Dealing with deprecated dependencies and un-updated modules to work with PyTest and to configure Travis CI was no small feat! Our team jumped up and high-fived each other the day our build successfully passed Travis CI. We soon discovered that <font color="green">green</font> is a beautiful color. We realize how valuable testing is within a codebase. Good testing helps ensure robust code and provides a form of documentation to guide future developers.
+
+ ![Travis Build Passing](/img/blog/2016/hackbrighters_build_passing.png)<i><font color="green">Green </font>is truly beautiful in the world of testing.</i>
```

Comment:
the html tags here are in the wrong order, making it look a little wonky. the cleanest way would be to replace it with:

`<span class ="image-credits"><font color="green">Green</font> is truly beautiful in the world of testing. (Image: Team Hackbrighters)</span>`

This will set everything to the image credits style (on the same line) and keep the green word green.
Otherwise, you can use the `<span class>` above for the image credits only — it would work as well.